apiserver/allfacades.go

```diff
@@ -123,6 +123,7 @@ func AllFacades() *facade.Registry {
 	reg("Application", 2, application.NewFacade)
 	reg("Application", 3, application.NewFacade)
 	reg("Application", 4, application.NewFacade)
+	reg("Application", 5, application.NewFacade) // UpdateSeries
```

Comment:
This is not the preferred way to do it - the fact that it was done this way previously was a mistake. See StorageProvisionerAPIv3 and StorageProvisionerAPIv4 for the right way.